core/src/json.ts

```diff
@@ -17,17 +17,14 @@
  * limitations under the License.
  */
 
-import BoltProtocolV2 from '../../../bolt-connection/lib/bolt/bolt-protocol-v2'
-import utils from '../test-utils'
-
-describe('#unit BoltProtocolV2', () => {
-  beforeEach(() => {
-    jasmine.addMatchers(utils.matchers)
-  })
-
-  it('should return correct bolt version number', () => {
-    const protocol = new BoltProtocolV2(null, null, false)
-
-    expect(protocol.version).toBe(2)
-  })
-})
+/**
+ * Provides a interface to serialize values which doesn't support serialization by JSON.stringify
+ * @private
+ * @param val The value to be serialized as JSON
+ * @returns the json
+ */
+export function stringify (val: any) {
+  return JSON.stringify(val, (_, value) =>
+    typeof value !== 'bigint' ? value : `${value}n`
```

Comment:
Personally I find it clearer to read ternaries as `typeof value === 'bigint' ? `${value}n` : value`

Reply:
It makes sense, It was my `C/C++` accent.